ui/shared/status/StatusChatInput.qml

```diff
@@ -209,6 +226,26 @@ Rectangle {
                 emojiSuggestions.close()
             }
         }
+
+        let shiftOrCtrlModifier = (event.modifiers & Qt.ShiftModifier) || (event.modifiers & Qt.ControlModifier) || (event.modifiers & Qt.MetaModifier)
+        let arrowKeys = event.key == Qt.Key_Left || event.key == Qt.Key_Right || event.key == Qt.Key_Up || event.key == Qt.Key_Down
+
+        if (event.key !== Qt.Key_Backspace &&
+            event.key !== Qt.Key_Enter &&
+            event.key !== Qt.Key_Return &&
+            event.key !== Qt.Key_Tab &&
+            event.key !== Qt.Key_Shift &&
+            event.key !== Qt.Key_Ctrl &&
+            event.key !== Qt.Key_Copy &&
+            event.key !== Qt.Key_Cut &&
+            event.key !== Qt.Key_Select &&
+            event.key !== Qt.Key_Undo &&
+            event.key !== 16777249 &&
```

Comment:
I couldn't manage to capture `CMD` on Mac. The QML Docs say that `Key_Ctrl` will cover it, but apparently it doesn't. Had to hardcode the key code here to preserve "select all", "copy", "paste" etc. shortcuts